go/client/fork_server_windows.go

```diff
@@ -47,6 +49,10 @@ func spawnServer(g *libkb.GlobalContext, cl libkb.CommandLine, forkType keybase1
 	attr := syscall.ProcAttr{
```

Comment:
There is a comment above that call that says
```
// On 'nix this would include Setsid: true, which means
// the new process inherits the session/terminal from the parent.
// This is default on windows and need not be specified
```

maybe it could be changed to explain new behavior.